### DISCUSSION TITLE
Changing evidence autocompletion to be case-insensitive

### DISCUSSION THREAD

**Discussion by fz42net** [GhostManager/Ghostwriter#440] [Ideas]
First off, thanks for such a great product. It's been a joy to work with!

One of the common situations that I find myself in is inserting evidence with the autocompletion feature inside of the TinyMCE editor. This functionality is case sensitive, which is different from the other search functionalities inside of GW (Clients, Findings, Reports, Projects, etc.) This adds friction to reporting and increases the number of keystrokes as I'm trying to filter and insert evidence.

This is a simple example, but I have 3 pieces of evidence associated with a finding, that all contain different case variations of the word "docker". 
![EvidenceFiles](https://github.com/GhostManager/Ghostwriter/assets/75886894/4c903fa6-3717-466c-b618-93a1cfbfffad)

The following two screenshots demonstrate the current behavior, which requires me to backspace and change the case of the search term:
![DefaultBehaviorLowercase](https://github.com/GhostManager/Ghostwriter/assets/75886894/a12f822e-6bef-49dd-b74d-404997b0f289)
![DefaultBehaviorUppercase](https://github.com/GhostManager/Ghostwriter/assets/75886894/981c650f-21a7-4a28-9f28-f873b3affdf6)

Finally, the proposed idea is to make the autocomplete/search case-insensitive, which is demonstrated in the following screenshot:
![NewBehavior](https://github.com/GhostManager/Ghostwriter/assets/75886894/4f7d4fd0-0a07-4768-8374-ce4bafd874a3)

While this is a PEBKAC issue by not consistently applying case/formatting to my filenames, I imagine I'm not alone. I believe this would be a quality of life improvement that others could benefit from. 

It's a simple change and is visible in the following commit (https://github.com/fatzombi/Ghostwriter/commit/ba27375e7c19719aed57aab3680fe08642029e60). I'm no JS guru, so I'm not sure if there's a better way than what I'm doing, as indexOf is case sensitive.

Let me know if this is anything that could be included by default and needs an Issue created and Pull request submitted, or if it's small enough that a PR isn't necessary, or if you're not interested in changing this behavior.

**Comment by chrismaddalena** [comment]
Hey @fatzombi, I missed this being opened under Discussions. I'll include this change in the release I'm tagging today. Thanks for bringing it up!

Let me know if there's anything else. I'll leave the topic open.
